2019-Unnamed-NeoPixel/2019-Unnamed-NeoPixel.ino

```diff
@@ -0,0 +1,167 @@
+#include <Adafruit_NeoPixel.h>
+
+#define WaitPeriod 200
+#define TimeoutPeriod 20
+
+//Static loop variables:
+  #define SLOOP(min, maxv) static int i = (min); static int j = (min); if (i >= maxv) { i = (min); } int max = (maxv);
+  #define ENDSLOOP(min,maxv) j = i; if (i < (maxv)) {i++;}
+
+#define PIN 4
+#define NUM_PIXELS 12
+
+#define RED Adafruit_NeoPixel::Color(255,0,0)
+#define GREEN Adafruit_NeoPixel::Color(0, 255,0)
+#define BLUE Adafruit_NeoPixel::Color(0,0,255)
+#define YELLOW Adafruit_NeoPixel::Color(255,255,0)
+#define OFF Adafruit_NeoPixel::Color(0,0,0)
+
+Adafruit_NeoPixel strip = Adafruit_NeoPixel(NUM_PIXELS, PIN, NEO_GRB + NEO_KHZ800);
+int numPixels = strip.numPixels();
+void forwards()
```

Comment:
The patterns could use better names (e.g. this looks like it's `disabled`). The patterns _other_ than `forwards*/backwards*` have fine names.